infrastructure/ansible/roles/traffic_ops/templates/cdn.conf.j2

```diff
@@ -75,6 +75,8 @@
       "read_header_timeout" : {{ to_go_read_header_timeout }},
       "read_timeout" : {{ to_go_read_timeout }},
       "write_timeout" : {{ to_go_write_timeout }},
-      "supported_ds_metrics" : {{ to_go_supported_ds_metrics | to_json }}
+      "supported_ds_metrics" : {{ to_go_supported_ds_metrics | to_json }},
+      "plugins": {{ to_plugin_config.keys() | list | to_json }},
+      "plugin_config" : "{{ to_plugin_config | to_nice_json(indent=2) }}"
```

Comment:
Remove the quotes around this interpolation or it will try to treat the json as a string